.github/workflows/node.js.yml

```diff
@@ -8,15 +8,31 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
-
+env:
+  DOCKER_TAG: 0.19.1
```

Comment:
This seems brittle. Why do we need to specify this? Can we instead have an override + default value to latest?

Reply:
Yea I debated between `latest` and `0.19.1`. `latest` works fine, I just selected `0.19.1` so that our tests wouldn't block all our PRs if core put in a breaking change... Really we should have some tests running against `edge` in `core` so that they break if something bad is checked in.
I'll change this to `latest` for now (since it should be latest stable version).